packages/core/src/docdb/explorer/docdbNode.ts

```diff
@@ -24,10 +26,11 @@ export class DocumentDBNode extends AWSTreeNodeBase {
     public override async getChildren(): Promise<AWSTreeNodeBase[]> {
         return await makeChildrenNodes({
             getChildNodes: async () => {
-                return Promise.resolve([]) //TODO: Get clusters from region
+                const clusters = await this.client.listClusters()
+                return clusters.map(cluster => new DBClusterNode(cluster))
             },
-            getNoChildrenPlaceholderNode: async () => new PlaceholderNode(this, '[Nothing to see here yet.]'),
-            //new PlaceholderNode(this, localize('AWS.explorerNode.docdb.noClusters', '[No Clusters found]')),
+            getNoChildrenPlaceholderNode: async () =>
+                new PlaceholderNode(this, localize('AWS.explorerNode.docdb.noClusters', '[No Clusters found]')),
```

Comment:
Do we need to add any resources with the actual localization?

Reply:
Good question to be answered by AWS.